prepare/processors/to_list_by_hyphen.py

```diff
@@ -0,0 +1,39 @@
+from unitxt import add_to_catalog
+from unitxt.operator import SequentialOperator
+from unitxt.operators import RemoveValues
+from unitxt.processors import ToListByHyphenSpace
+
+add_to_catalog(
+    SequentialOperator(
+        steps=[
+            ToListByHyphenSpace(field="prediction", process_every_value=False),
```

Comment:
```suggestion
         SplitStrip(delimiter="- ", strip_every_element=True, process_every_value=False),
```
I think it it is cleaner to have it like this rather then define a new class.

Reply:
Thank you for the review! Yes, I agree, and in fact, when I personally used this processor, I implemented it in the same file. However, since the implementation of [to_list_by_comma](https://github.com/IBM/unitxt/blob/main/prepare/processors/to_list_by_comma.py) defined a class and separated the files, I followed that approach. I don't have a strong preference, so we could either consolidate this processor into one file, refactor to_list_by_comma, or keep it as is following the current implementation of to_list_by_comma. Which option do you prefer?